libs/api/domains/air-discount-scheme/src/lib/models/flight.model.ts

```diff
@@ -0,0 +1,35 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+import { UserInfo as TUserInfo } from '@island.is/air-discount-scheme/types'
+import { User } from './user.model'
+import { FlightLeg } from './flightLeg.model'
+
+@ObjectType()
+export class UserInfo implements TUserInfo {
+  @Field()
+  gender!: 'kk' | 'kvk' | 'hvk'
```

Comment:
This will be fixed upon later database migration and type interaction with Loftbrú